src/index.js

```diff
@@ -3,10 +3,21 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App/App';
 import { BrowserRouter } from 'react-router-dom';
+import { ApolloProvider, ApolloClient, InMemoryCache } from '@apollo/client';
 
-const router =
-  <BrowserRouter>
-    <App />
-  </BrowserRouter>;
+const client = new ApolloClient({
+  uri: "https://frozen-waters-94259.herokuapp.com/graphql",
+  cache: new InMemoryCache()
+})
 
-ReactDOM.render(router, document.getElementById('root'));
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
Could make this a variable like before "router = " but could be "apolloRouter = "